buildAndReleaseTask/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yavt",
-  "version": "1.6.1",
+  "version": "1.7.0",
```

Comment:
Could you please also update the version in `vss-extension.json`? ...yes, there are three places where it's required 😆